### ISSUE TITLE
Rolling multiple sets of dice at the same time can result in dice overlapping

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#206]
For example, adding four 1s with the Thief's limit ability and using a Lockpick while the dice are still entering the pool results in dice overlapping.

**Comment by TerryCavanagh** [comment]
```
with robot, first turn of the game, he calculated a 1, then accidentally calculated twice 
by clicking very very fast. Then got an error, but he only had a 1 and a 4. 
When he moved the 4 aside and there was a 5 under it
aka the second and third dice spawned in the exact same spot cause he clicked too fast
```

**Comment by pace212** [comment]
This is easy to reproduce with Jester's Snap. I used Snap to activate 2 High Voltage cards while the enemy was shocked. I got two 4s back and they lerped onto the screen overlapping. In another run I used Snap on two Bumps. See the 6 and the 1 here?

![screenshot 2019-03-03 14 50 34](https://user-images.githubusercontent.com/38589027/53703182-b9679080-3dc3-11e9-9489-b405420f3241.png)

Those are actually two 6s and two 1s.
